Tools/CMake/AMReXParallelBackends.cmake

```
@@ -31,7 +31,7 @@ endif ()
 #
 if (AMReX_OMP)
    set(_omp_comps CXX)
-   if (AMReX_FORTRAN_INTERFACES)
+   if (AMReX_FORTRAN_INTERFACES OR AMReX_FORTRAN)
```

```suggestion
   if (AMReX_FORTRAN)
```
Testing for ``AMReX_FORTRAN_INTERFACES`` is redundant if we also test for ``AMReX_FORTRAN``